tonic/Cargo.toml

```diff
@@ -42,44 +42,43 @@ prost = ["prost1", "prost-derive"]
 # harness = false
 
 [dependencies]
-bytes = "0.5"
+bytes = "1"
 futures-core = { version = "0.3", default-features = false }
 futures-util = { version = "0.3", default-features = false }
 tracing = "0.1"
 http = "0.2"
-base64 = "0.12"
+base64 = "0.13"
 
 percent-encoding = "2.0"
 tower-service = "0.3"
-tokio-util = { version = "0.3", features = ["codec"] }
-async-stream = "0.2"
-http-body = "0.3"
-pin-project = "0.4.17"
+tokio-util = { version = "0.6", features = ["codec", "io"] }
+async-stream = "0.3"
+http-body = "0.4"
+pin-project = "1"
 
 # prost
-prost1 = { package = "prost", version = "0.6", optional = true }
-prost-derive = { version = "0.6", optional = true }
+prost1 = { package = "prost", version = "0.7", optional = true }
+prost-derive = { version = "0.7", optional = true }
 
 # codegen
-async-trait = { version = "0.1.13", optional = true }
+async-trait = { version = "0.1.42", optional = true }
 
 # transport
-hyper = { version = "0.13.4", features = ["stream"], optional = true }
-tokio = { version = "0.2.13", features = ["tcp"], optional = true }
-tower = { version = "0.3", optional = true}
-tower-make = { version = "0.3", features = ["connect"] }
-tower-balance =  { version = "0.3", optional = true }
-tower-load = { version = "0.3", optional = true }
-tracing-futures = { version = "0.2", optional = true }
+hyper = { version = "0.14.1", features = ["full"], optional = true }
+tokio = { version = "1.0.1", features = ["net"], optional = true }
+tokio-stream = { version = "0.1", optional = true }
+tower = { git = "https://github.com/tower-rs/tower", features = ["balance", "buffer", "discover", "limit", "load", "make", "timeout", "util"], optional = true }
```

Comment:
We will need a tower release before we can get tonic out for sure.

Reply:
Tower 0.4.1 is out now for Tokio 1.